Cargo.toml

```diff
@@ -12,7 +12,9 @@ repository = "https://github.com/CraneStation/target-lexicon"
 edition = "2018"
 
 [features]
-default = []
+default = ["rust_1_40"]
+# When enabled, use features that are only available with 1.40 or later.
+rust_1_40 = []
```

Comment:
You could use a build script. It is often very hard to disable default features, as everyone crate that depends on target-lexicon has to use `default-features = false`.

Reply:
Good idea!